cloudformation/al_security_ids_full.template

```diff
@@ -45,7 +45,7 @@
         "InstanceType": {
             "Description": "Select the instance type to launch IDS security appliances",
             "Type": "String",
-            "Default": "c5.large",
+            "Default": "c5.xlarge",
```

Comment:
@Chithrak07 I just noticed this - we've doubled the size of all these instances, which is causing overages for all the projects using them. 

Is this a typo? or is the new default actually xlarge?

Reply:
yes, the default is not c5.xlarge. https://docs.alertlogic.com/deploy/aws-manual-pro-ent.htm#Deployscanningappliances
Should I revert this?

Reply:
We need to find out from alert logic if xlarge is actually necessary.

Reply:
Support ticket created.